_template/api/proto/buf.yaml

```diff
@@ -6,11 +6,14 @@ deps:
   - buf.build/envoyproxy/protoc-gen-validate
   - buf.build/googleapis/googleapis
 {{- if .Extensions.grpc.grpcGateway }}
-  - buf.build/grpc-ecosystem/grpc-gateway
+  - buf.build/gnostic/gnostic
 {{- end }}
 lint:
   use:
     - DEFAULT
   except:
     - RPC_RESPONSE_STANDARD_NAME
     - RPC_REQUEST_RESPONSE_UNIQUE
+{{- if .Extensions.grpc.grpcGateway }}
+    - IMPORT_NO_WEAK # required by gnostic for now. this way gen-go will not import _ "" broken packages
```

Comment:
Is it needed to exclude that globally if you also add the `// buf:lint:ignore IMPORT_NO_WEAK` directive in the proto file?